src/serac/infrastructure/debug_print.hpp

```diff
@@ -16,14 +16,42 @@
 #include "serac/infrastructure/memory.hpp"
 #include "serac/numerics/functional/element_restriction.hpp"
 
```

Comment:
```suggestion
#include <cxxabi.h>
```

Reply:
`abi::__cxa_demangle()` is provided by `cxxabi.h`. We're getting it here from some transitive include. I'd put it in explicitly in case someone upstream refactors.

Reply:
ah good catch.. forgot to copy/paste all my includes from my smaller example.